doc/src/manual/performance-tips.md

```diff
@@ -477,6 +477,12 @@ Here, we happened to know that the first element of `a` would be an [`Int32`](@r
 an annotation like this has the added benefit that it will raise a run-time error if the
 value is not of the expected type, potentially catching certain bugs earlier.
 
+A slightly more generic way to annotate the type of `x` is `x = convert(Int32,a[1])::Int32`.  The
```

Comment:
Perhaps rephrase this to "A slightly more generic way to annotate the type of `x` is shown by the example of...". It took me a second to realize you are referring to the `::Int32` bit and the fact that `convert` is involved is somewhat incidental (if I've understood correctly).